pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -84,6 +85,66 @@ pub struct IndexPart {
 
     #[serde(rename = "metadata_bytes")]
     pub metadata: TimelineMetadata,
+
+    #[serde(default)]
+    pub(crate) lineage: Lineage,
```

Comment:
Lineage basically duplicates things which should be part of the TimelineMetadata.

In future PR, I want to get rid of the bytes serialization, as the decimal byte makes `jq .` pretty annoying to print. We most likely can never get rid of "ingesting TimelineMetadata" bytes but at least we can fix the future ones.

Reply:
#7693 and it's follow-up #7699 for this idea.